pages/portfolio.html

```diff
@@ -4,9 +4,59 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>My Work</title>
+  <link href="/stylesheets/style.css" rel="stylesheet">
+  <link href="/stylesheets/portfolio-style.css" rel="stylesheet">
 </head>
+
 <body>
+  <main class="container">
+    <header><h1>tiffini hyatt</h1></header>
+
+    <nav>
+      <menu class="nav-links">
+        <li id="home"><a href="pages/index.html"><h4>home</h4></a></li>
+        <li id="portfolio"><a href="pages/portfolio.html"><h4>work</h4></a></li>
+        <li id="about"><a href="pages/about.html"><h4>about me</h4></a></li>
+      </menu>
+    </nav>
+
+    <div class="projects">
+      <div id="project-1">
+        <img src="/assets/checklist-icon.png"
+          alt="A simple line drawing of a piece of paper with three checkboxes
+          next to three lines representing text.">
+        <h3><a class="project-title" href="https://github.com/tiffinihyatt/task-list-api">To-Do List API</a></h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do 
+          eiusmod tempor incididunt ut labore.</p>
+        <p><strong>Tech: </strong>Python, Flask, PostgreSQL</p>
```

Comment:
The `strong` element is intended to be used to denote content that is important and being emphasized, it shouldn't be used for its default styling if the meaning is not present. If we want bold text without implying extra importance or emphasis, we should use `p` or `span` elements (`span` in this particular case) and style them as needed.